docs/examples/widgets/sparkline_basic.py

```diff
@@ -0,0 +1,19 @@
+from textual.app import App, ComposeResult
+from textual.widgets._sparkline import Sparkline
+
+data = [1, 2, 2, 1] + [1, 4, 3, 1] + [1, 8, 8, 2]  # (1)!
```

Comment:
I feel that creating this list of 12 data points, by joining 3 lists, confuses the whole "bucketing" thing if you look at the code. Normally folk wouldn't create a list this way so the fact that it's done like this seems like it's significant or a callout to something I need to be aware of...

I'd just make this one single list to better-emphasise the fact that the "bucketing" is unrelated to the content of the list.

Reply:
Thanks for the suggestion.